src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -167,6 +168,13 @@ private function findClasses(string $namespace, string $pattern, array $excludeP
             }
         }
 
+        if (80000 <= \PHP_VERSION_ID) {
+            $parseDefinition = new \ReflectionMethod(YamlFileLoader::class, 'parseDefinition');
```

Comment:
Reusing the implementation in `YamlFileLoader` provides compat with nice syntaxes allowed by the loader, including references.
This won't provide support for tagged arguments, but I don't think we need to provide support for them, for now at least.